Update for 2025 Regulations (Delegate Handbook) (#451)
thewca/wca-documents: edudoc/delegate-handbook/delegate-handbook.md

   <td>Competitor did not sign for an attempt</td>
   <td>
   If the competition is not over, you may call over the competitor to sign for the attempt after the fact via [Regulation A7c5](https://www.worldcubeassociation.org/regulations/#A7c5). Otherwise, enter the attempt as a DNF. Video or photographic evidence may not be used in lieu of a missing signature.</br></br>
   Entering a DNF attempt often prompts the competitor to appear once they check their results on WCA Live. You may still allow the competitor to sign for the attempt at your discretion.</br></br>

>    For inexperienced competitors, additional lenience is permitted by [Regulation A7c8](https://www.worldcubeassociation.org/regulations/#A7c8) and missing signaturs may be added at any time at your discretion. 

Author: I think it's better to add the reference there rather than duplicating and have committed there instead.

>    For inexperienced competitors, additional lenience is permitted by [Regulation A7c8](https://www.worldcubeassociation.org/regulations/#A7c8) and missing signaturs may be added at any time at your discretion. 

Reviewer: Ok, I removed the reference from the suggestion, but there's still a typo.

[Regulation 3j](https://www.worldcubeassociation.org/regulations/#3j) defines what is considered reasonable wear for a puzzle to be used in a competition. To increase the consistency in how reasonable wear is evaluated, the WRC has created a [Regulation 3j Visual Guide](https://drive.google.com/file/d/1m6THsA8fXRN7QFM4ApJbm6eVODKGbMLx/view) to assist Delegates in making rulings regarding wear. Familiarize yourself with the examples included in the guide. Some cases of wear will not perfectly match the provided examples. Make your best judgment by extrapolating from the given examples. If there is doubt about the legality of a puzzle, it is safest to request that a competitor use a different puzzle to prevent their results from being retroactively removed.

> If an illegal puzzle is found during a round, any completed attempts may be replaced with extras at your discretion via [Regulation 3k2a](https://www.worldcubeassociation.org/regulations/#3k2a). If the round has already been completed, such as may be the case if an illegal puzzle is discovered to have been used in Round 1 during Round 2, these attempts may not be replaced with extras (exception: for inexperienced competitors, extras maybe given after the round is complete using [Regulation A7g](https://www.worldcubeassociation.org/regulations/#A7g), but may not be given if a subsequent round has started due to [Regulation 9l](https://www.worldcubeassociation.org/regulations/#9l)).

Reviewer: ```suggestion
If an illegal puzzle is found during a round, any completed attempts may be replaced with extras at your discretion via [Regulation 3k2a](https://www.worldcubeassociation.org/regulations/#3k2a). If the round has already been completed, such as may be the case if an illegal puzzle is discovered to have been used in Round 1 during Round 2, these attempts may not be replaced with extras. Exception: for inexperienced competitors, extras may be given after the round is complete using [Regulation A7g](https://www.worldcubeassociation.org/regulations/#A7g), but may not be given if a subsequent round has started, due to [Regulation 9l](https://www.worldcubeassociation.org/regulations/#9l).
```

### Regulation A7g

> Many new competitors will inadvertently break the Regulations due to their unfamiliarity with the Regulations and processes of a WCA competition or an event's specific regulations. In these cases, you are always encouraged to use [Regulation A7g](https://www.worldcubeassociation.org/regulations/#A7g) to issue an extra to these competitors. An inexperienced competitor defined by A7g may not necessarily be new to WCA competitions, such as a competitor who is returning to WCA competitions after a long time or a competitor who is unfamiliar with a specific event ([Guideline A7g++++](https://www.worldcubeassociation.org/regulations/guidelines.html#A7g++++)). For example, a blindfolded-focused competitor who is newly competing in other speedsolving events may not be used to the judge lifting the cube cover for them. Use your discretion when issuing extras for incidents like these.

Author: ```suggestion
Many new competitors will inadvertently break the Regulations due to their unfamiliarity with the Regulations and processes of a WCA competition or an event's specific regulations. In these cases, you are always encouraged to use [Regulation A7g](https://www.worldcubeassociation.org/regulations/#A7g) to issue an extra to these competitors. An inexperienced competitor may not necessarily be new to WCA competitions, such as a competitor who is returning to WCA competitions after a long time or a competitor who is unfamiliar with a specific event ([Guideline A7g++++](https://www.worldcubeassociation.org/regulations/guidelines.html#A7g++++)). For example, a blindfolded-focused competitor who is newly competing in other speedsolving events may not be used to the judge lifting the cube cover for them. Use your discretion when issuing extras for incidents like these.
```

  <tr>
   <td>Competitor is using a device with a front-facing screen</td>
   <td>
   The attempt must be changed to a DNF result.</br></br>

>    Any previous attempts in which the device was used must also be changed to DNF. The penalty for previous attempts may be lifted at your discretion if you believe the competitor did not gain assistance from the screen, but you must not allow use of the screen to continue.</br></br>

Reviewer: ```suggestion
   Any previous attempts in which the device was used must also be changed to DNF. The penalty for previous attempts may be lifted at your discretion if you believe the competitor did not get assistance from the screen, but you must not allow them to keep using the screen in such way.</br></br>
```
Feel free to ignore this suggestion.

  <tr>
   <td>Timer malfunction</td>
   <td>
   If the time is strictly **below** 0.06 seconds, an extra **must** be issued according to [Regulations A6b1](https://www.worldcubeassociation.org/regulations/#A6b1) unless the competitor stopped the timer deliberately or caused the timer to stop by starting it in an usual manner.</br></br>
   Make note of the timer. If it continues to malfunction, remove or replace the timer with one that is functioning properly. It is recommended to mark this timer in some way so that it is not re-introduced to the competition area without being tested first.</br></br>

>    If the time is equal to 0.06 seconds or higher, or below 0.06 seconds and the competitor stopped the timer with the puzzle unsolved, the result is a DNF.

Author: ```suggestion
   If the time is equal to 0.06 seconds or higher, or below 0.06 seconds and the competitor stopped the timer, the result is a DNF.
```
